backend/substrapp/tests/views/test_views_authentication.py

```diff
@@ -102,11 +102,44 @@ def test_obtain_token(self):
                                     {'username': 'foo', 'password': 'baz'}, **self.extra)
         self.assertEqual(response.status_code, 400)
 
+        response = self.client.post('/api-token-auth/',
+                                    {'username': 'foo', 'password': 'bar'}, **self.extra)
+        self.assertEqual(response.status_code, 200)
+        token_old = response.json()['token']
+        self.assertTrue(token_old)
+
+        # token should be update after a second post
         response = self.client.post('/api-token-auth/',
                                     {'username': 'foo', 'password': 'bar'}, **self.extra)
         self.assertEqual(response.status_code, 200)
         token = response.json()['token']
-        self.assertTrue(token)
+        self.assertTrue(token_old)
```

Comment:
I think this should still be `self.assertTrue(token)`

Reply:
Yes !